db/migrate/20220718072425_create_tasks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title
+      t.boolean :status
```

Comment:
Please also add nullable constraint and default value to it.

Reply:
Ok, I will add them together with the controller in the next PR

```
class CreateTasks < ActiveRecord::Migration[7.0]
  def change
    create_table :tasks do |t|
      t.string :title, null: false, default: ''
      t.boolean :status, null: false, default: false

      t.timestamps
    end
  end
end
```